spring-cloud-azure-samples/spring-cloud-azure-servicebus-integration-sample/README.adoc

```diff
@@ -19,7 +19,7 @@ Make sure `my.azureauth` is encoded with UTF-8.
 
 2. Put auth file under `src/main/resources/`.
 
-3. Create https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-create-namespace-portal[Azure Service Bus]
+3. Create https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-create-namespace-portal[Azure Service Bus Namespace], then create service bus queue and topic.
```

Comment:
Create service bus queue and topic. Please follow `link`. Same as other sample doc.